RadixWallet/Features/AppFeature/Overlay/InfoLinkSheet+Reducer.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+// MARK: - InfoLinkSheet
+public struct InfoLinkSheet: FeatureReducer {
+	public struct State: Sendable, Hashable, Identifiable {
```

Comment:
Why do we need this to implement `Identifiable`?

Reply:
No, removed.